scripts/sql/213_cosign_plugin.up.sql

```diff
@@ -0,0 +1,89 @@
+INSERT INTO plugin_metadata (id,name,description,type,icon,deleted,created_on,created_by,updated_on,updated_by)
+VALUES (nextval('id_seq_plugin_metadata'),'Cosign v1.0.0','This plugin is used to Cosign to sign the docker images created in CI.','PRESET','https://github.com/jatin-jangir-0220/test/blob/main/CDBA2B21-4D52-4A1A-8BF4-C9F66B9CF2FF.png?raw=true',false,'now()',1,'now()',1);
```

Comment:
Please fix grammatical errors in description

Reply:
Also, change plugin icon url to devtron repo main branch

Reply:
done